test/stat_tracker_test.rb

```diff
@@ -161,4 +161,28 @@ def test_it_can_get_team_with_fewest_tackles
     assert_equal "Atlanta United", @stat_tracker.fewest_tackles("20132014")
     assert_equal "Orlando City SC", @stat_tracker.fewest_tackles("20142015")
   end
+
+  def test_it_can_return_best_offense
+    assert_equal "Reign FC", @stat_tracker.best_offense
+  end
+
+  def test_it_can_return_worst_offense
+    assert_equal "Utah Royals FC", @stat_tracker.worst_offense
+  end
+
+  def test_it_can_return_highest_scoring_visitor
+    assert_equal "Reign FC", @stat_tracker.highest_scoring_visitor
+  end
+
+  def test_it_can_return_lowest_scoring_visitor
+    assert_equal "Utah Royals FC", @stat_tracker.lowest_scoring_visitor
+  end
```

Comment:
The spec harness has the highest scoring visitor as "FC Dallas" and lowest scoring visitor as "San Jose Earthquakes". We may want to double check these methods.

Reply:
agreed, we can get on a call this week and dive into these methods to see if we can figure out where things are getting off.

Reply:
Also noticing these tests take a very long time to run, we will need to look into how to make these methods more efficient.